spec/system/admin/tenants_spec.rb

```diff
@@ -40,6 +40,30 @@
       expect(page).to have_current_path root_path
       expect(page).to have_link "Sign in"
     end
+
+    scenario "Updates new tenant default user with the main tenant administrator login credentials" do
+      allow(Rails.env).to receive(:test?).and_return(false)
```

Comment:
There might side effects if we tell Rails we aren't in the test environment when we actually are :thinking:. There are also a couple of tests in this file which are currently failing for the same reason. What do you think about removing this line and updating the `seeds.rb` instead?

```ruby
if Administrator.count == 0 && (!Rails.env.test? || !Tenant.default?)
```

This would be an option, but if you find a better one, go for that one!

Reply:
Yes, it was a crazy idea, we wanted to confirm that this would break something, and yes, it did. Finally, got your suggestion while we find a better way to deal with default users.